### PR TITLE
drop-user

### PR DESCRIPTION
**Pull request by aarturoc**
This PR has info about drop-user.

[Preview here](https://github.com/GalileiLearning/glowing-octo-dollop/tree/drop-user/9)

### CODE REVIEW

File: 9/README.md [hunk 1]
```diff
@@ -1 +1,36 @@
-# Drop user
\ No newline at end of file
+# Drop user
+As an admin, I want to drop an user.
+
+### Actors
+Admin
+
+### Preconditions
+The admin needs the user corporate email so he can drop the user.
```
Comment (andymotagl) [inline 9/README.md:8]: The user must exists?

File: 9/README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+3. The user already exist.
+
+### Flow Diagram
+
+```mermaid
+flowchart TD
+    A[Start] --> B{is the corporate email correct?}
+    B -- Yes --> D{has the corporative email the same domain as the admin's?}
+    D -- Yes --> E{does the user exist?}
+    E -- Yes --> H[Returns user already exist]
```
Comment (andymotagl) [inline 9/README.md:28]: I guess same as the comment above

**Review by andymotagl — COMMENTED** [review]
Some comments

**Comment by aarturoc** [comment]
Closing this PR

**Comment by aarturoc** [comment]
I update the file readme with the suggestion that you make me, can you check it?

File: 9/README.md [hunk 3]
```diff
@@ -1 +1,37 @@
-# Drop user
\ No newline at end of file
+# Drop user
+As an admin, I want to drop an user.
+
+### Actors
+Admin
+
+### Preconditions
+1. User most be added first.
```
Comment (andymotagl) [inline 9/README.md:8]: must be exist?

File: 9/README.md [hunk 4]
```diff
@@ -1 +1,37 @@
...
+2. The admin needs the user corporate email.
```
Comment (andymotagl) [inline 9/README.md:9]: Technically speaking the system would need the user identifier to be deleted so, I wonder if only with the first precondition implies this one?

File: 9/README.md [hunk 5]
```diff
@@ -1 +1,37 @@
...
+
+### Scenarios
+
+#### Success
+Admin drop the new user.
```
Comment (andymotagl) [inline 9/README.md:14]: Remove new

File: 9/README.md [hunk 6]
```diff
@@ -1 +1,37 @@
...
+
+#### Failure
+
+1. The corporative email isn't correct.
```
Comment (andymotagl) [inline 9/README.md:18]: About the comment above regarding providing the identifier, would this scenario will be avoided?

File: 9/README.md [hunk 7]
```diff
@@ -1 +1,37 @@
...
+3. The user does not exist.
+
+### Flow Diagram
+
+```mermaid
+flowchart TD
+    A[Start] --> B{is the corporate email correct?}
+    B -- Yes --> D{has the corporative email the same domain as the admin's?}
+    D -- Yes --> E{does the user exist?}
+    E -- Yes --> H[Returns user successfully droped]
```
Comment (andymotagl) [inline 9/README.md:29]: Removed instead of dropped

**Review by andymotagl — COMMENTED** [review]
Great diagram out there!
Just pointed out few things but in general, good job

**Comment by aarturoc** [comment]
i took the suggestion  and made the change, what do you think?
[Preview here](https://github.com/GalileiLearning/glowing-octo-dollop/tree/drop-user/9)

**Review by andymotagl — APPROVED** [review]
Great!

**Pull request merged**